src/demo/DemoTestSearchService.java

```diff
@@ -172,6 +194,149 @@ public static void testTypesCombinations() throws MambuApiException {
 
 	}
 
+	// Test API to GET entities by On The Fly Filter
+	private static void testSearchEntitiesByFilter() throws MambuApiException {
```

Comment:
Could you please add a few more test cases for:
1) multiple contraints in one filter
2) custom field filter
3) date filter using between to see if "second value" works